### PR TITLE
Disable coverage

### PR DESCRIPTION
**Pull request by rikhuijzer**
Let's see whether the tests pass within time again when disabling coverage.

### CODE REVIEW

**Comment by rikhuijzer** [comment]
One sort of deprecation warning visible in the ubuntu output for Julia 1.6
```
┌ Warning: the `normalize` keyword argument will be false by default in future releases: set it explicitly to silence this deprecation
│   caller = (::TuringGLM.var"#normal_model_ranef#16"{Int64, Int64, CustomPrior, Vector{String}, Int64, Vector{Int64}})(__model__::DynamicPPL.Model{TuringGLM.var"#normal_model_ranef#16"{Int64, Int64, CustomPrior, Vector{String}, Int64, Vector{Int64}}, (:y, :X, :predictors, :idxs, :n_gr, :intercept_ranef, :μ_X, :σ_X, :prior, :residual), (:predictors, :idxs, :n_gr, :intercept_ranef, :μ_X, :σ_X, :prior, :residual), (), Tuple{Vector{Int64}, Matrix{Float64}, Int64, Vector{Int64}, Int64, Vector{String}, Int64, Int64, CustomPrior, Float64}, Tuple{Int64, Vector{Int64}, Int64, Vector{String}, Int64, Int64, CustomPrior, Float64}, DynamicPPL.DefaultContext}, __varinfo__::DynamicPPL.UntypedVarInfo{DynamicPPL.Metadata{Dict{AbstractPPL.VarName, Int64}, Vector{Distribution}, Vector{AbstractPPL.VarName}, Vector{Real}, Vector{Set{DynamicPPL.Selector}}}, Float64}, __context__::DynamicPPL.SamplingContext{DynamicPPL.SampleFromUniform, DynamicPPL.DefaultContext, Random.MersenneTwister}, y::Vector{Int64}, X::Matrix{Float64}, predictors::Int64, idxs::Vector{Int64}, n_gr::Int64, intercept_ranef::Vector{String}, μ_X::Int64, σ_X::Int64, prior::CustomPrior, residual::Float64) at turing_model.jl:183
└ @ TuringGLM ~/work/TuringGLM.jl/TuringGLM.jl/src/turing_model.jl:183
```

**Comment by rikhuijzer** [comment]
I've also set the number of threads to two since Julia defaults to 1 in CI.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -9,19 +9,19 @@ concurrency:
   # Skip intermediate builds: always.
   # Cancel intermediate builds: only if it is a pull request build.
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+  cancel-in-progress: false
```
Comment (rikhuijzer) [inline .github/workflows/CI.yml:12]: Canceling runs in progress sounds useful, but in practice usually the following. One run finishes after running all the tests and detected a failing test, so the job reports a failure. All other jobs are at around 90% and get cancelled resulting in zero useful information. So, basically, the 90% of time spent on running tests is wasted when cancel-in-progress is `true`.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -9,19 +9,19 @@ concurrency:
...
-    timeout-minutes: 60
+    timeout-minutes: 90
     continue-on-error: ${{ matrix.version == 'nightly' }}
     strategy:
       fail-fast: false
       matrix:
         version:
-          - '1'
           - '1.6'
+          - '1.7'
```
Comment (storopoli) [inline .github/workflows/CI.yml:24]: Why not 1.8?

Comment (rikhuijzer) [reply .github/workflows/CI.yml:24]: Because I wanted to fix CI first

Comment (rikhuijzer) [reply .github/workflows/CI.yml:24]: Let's merge and switch to 1.8 in a separate PR?

Comment (storopoli) [reply .github/workflows/CI.yml:24]: Deal!

**Review by storopoli — APPROVED** [review]

**Pull request merged**